.circleci/config.yml

```diff
@@ -43,6 +43,15 @@ commands:
           name: Fetch dependencies with composer
           command: |
             composer install --no-interaction --optimize-autoloader
+  composer_tasks__edge_packages:
```

Comment:
For context: this was previously done like this:

* CI workflow scheduled to run M-F on the `development` branch.
* A job would then switch over the checked out branch (`development`), get composer to update to latest commits on repo branches, then commit + push back to GH.
* Really all we needed to do here was: in the workflow where we build the edge environment, set it to listen/pull on the edge branch (`D8NID-edge`), then fetch the latest commits, then push back. It's a bit simpler that way and avoids some mental gymnastics to understand what code is in operation at what stage in the pipeline.